### PR TITLE
Updated Backend tests for code coverage

### PR DESCRIPTION
**Pull request by cdubya25**

### CODE REVIEW

File: ETT-Backend.Tests/Repository/QueryGeneratorTest.cs [hunk 1]
```diff
@@ -31,5 +33,121 @@ public void ShouldReturnQueryForGettingAllEmployees()
...
+    {
+      var ettEmployeeObject = new JObject()
+      {
+        {"first_name", "anotherTest"},
+        {"last_name", "developer"},
+        {"employee_number", "Cincinnati"},
+        {"employee_email", "test@callibrity.com"}
+      };
+      var expectedEttEmployeeConflict = " ON CONFLICT (employee_number) DO UPDATE SET first_name=EXCLUDED.first_name,last_name=EXCLUDED.last_name,employee_email=EXCLUDED.employee_email";
+      var actualEttEmployeeConflit = QueryGenerator.OnConflict(ConstraintKeys["ett_employee"], ettEmployeeObject);
```
Comment (arielleferre23) [inline ETT-Backend.Tests/Repository/QueryGeneratorTest.cs:124]: typo :)

File: ETT-Backend.Tests/Services/EmployeeServiceTest.cs [hunk 2]
```diff
@@ -56,29 +97,59 @@ public void RetrieveEmployeeHoursInvalidEmail()
...
+      Assert.Equal(5.0, response[0].Growth.HoursUsed);
+      Assert.Equal(4.0, response[0].Growth.TotalGrowth);
+
+      Assert.Equal("lul", response[1].EmployeeId);
+      Assert.Equal(2.0, response[1].Billable.CurrentHours);
+      Assert.Equal(2.0, response[1].Billable.CurrentTarget);
+      Assert.Equal(2.0, response[1].Billable.TotalTarget);
+      Assert.Equal(0, response[1].Growth.HoursRemaining);
+      Assert.Equal(2.0, response[1].Growth.HoursUsed);
+      Assert.Equal(2.0, response[1].Growth.TotalGrowth);
```
Comment (arielleferre23) [inline ETT-Backend.Tests/Services/EmployeeServiceTest.cs:133]: Was there a reason to verify two db entries? Just curious

File: ETT-Backend.Tests/Repository/QueryGeneratorTest.cs (hunk 1 above)
Comment (cdubya25) [reply ETT-Backend.Tests/Repository/QueryGeneratorTest.cs]: fixed!

File: ETT-Backend.Tests/Services/EmployeeServiceTest.cs (hunk 2 above)
Comment (cdubya25) [reply ETT-Backend.Tests/Services/EmployeeServiceTest.cs:133]: the method was to retrieve all employees' metrics so I wanted to return more than one

**Review by arielleferre23 — DISMISSED** [review]

**Review by arielleferre23 — APPROVED** [review]

**Pull request merged**